### ISSUE TITLE
Need help with Essentials Shops (IDs Issue)

### ISSUE THREAD

**Issue by Faisal2Rock** [EssentialsX/Essentials#2666]
I have been searching for hours, and I am still completely confused. Basically, I am making a server for my friends and I, and we are using Essentials Buy/Sell signs in our economy. We set some of them up, and they do work. The problems arose when I needed to use Essentials signs for names that were very long and do not fit on signs (such as, mossycobblestone [doesn't fit]) spawners (non-pig). So, I tried to use minecraft numerical IDs and they do not work. I have heard on various threads that the IDs are discontinued or something. So, what am I supposed to do to sell items with long names, items with enchants, and spawners? I tried using https://github.com/EssentialsX/Essentials/blob/2.x/Essentials/src/items.csv#L5579, and https://github.com/EssentialsX/Essentials/blob/2.x/Essentials/src/items.json but these words are still too long for a sign to hold, and not all items are listed (again, enchanted stuff [unsafe enchants] and spawners). If someone could help it would be amazing. Thank you!

Basically, I need a way for Essentials Eco in 1.14 to be able to buy/sell mob spawners, long-named items, and unsafe enchanted items. If someone knows the solution, or other 1.14 plugins that may work, please, please, please, help me.

**Comment by Omio** [comment]
Potential workarounds in the meantime:
1) In your language settings, force the unicode font on.
2) Use kits with just the item in question.
3) Learn to use command blocks to change sign NBT data.

This is a limitation that EssentialsX can't help.

**Comment by mdcfe** [comment]
You can add manually add aliases to the `items.json`. We're planning to make this easier to do, but for now you can add a line with `"<alias>": "<material>"` to set up your own aliases.
